test/passport/controllers/two_factor_auth_controller_test.exs

```diff
@@ -0,0 +1,89 @@
+defmodule Passport.TwoFactorAuthControllerTest do
+  use Passport.Support.Web.ConnCase
+
+  describe "POST /account/confirm/tfa" do
+    test "can reset tfa_otp_secret_key", %{conn: conn} do
```

Comment:
What?

Reply:
Why would confirming reset the key

Reply:
Oh this is for starting a confirmation? What happens if I never trigger it? We don't need to manually trigger a mail send